app/javascript/components/analysis/components/show-analysis/styles.scss

```diff
@@ -1,11 +1,43 @@
 @import '~styles/settings';
 
 .c-show-analysis {
-  width: 100%;
-  padding: rem(20px);
-  min-height: rem(260px);
   height: 100%;
-  padding-bottom: rem(60px);
+  width: 100%;
+
+  .c-show-analysis-body {
```

Comment:
We don't want to be mixing `.c-` classname definitions as nested class names. Has the component structure changed?